### DISCUSSION TITLE
Portuguese Documetation of Vue.js 3

### DISCUSSION THREAD

**Discussion by nazarepiedady** [vuejs/docs#2072] [Show and tell]
Hi colleagues, I hope all is okay with you.
Today I announce the project that aims to translate to Portuguese, the
official Vue.js 3 documentation, it is a project that has already started and
is going well and is almost finished.

To people who do not know me, I am also the person behind the official
Portuguese version of the new [Nuxt.js 2 website](https://nuxtjs.org/pt/) and I have other works too,
such as the community translation of [Vue Test Utils 1 website](https://vue-test-utils-portuguese-docs.netlify.app/pt/), [Pinia website](https://pinia-docs-pt.netlify.app/),
to Portuguese. Also currently I am working on what will be the official
translation of the [Vite website](https://vite-docs-pt.netlify.app/) to Portuguese.

Here I left the link to the repository of the work:
https://github.com/nazarepiedady/vue3-docs-pt

I am available to do the necessary changes to adequate the repository to
the required requirements to be part of the [VueJS Translations](https://github.com/vuejs-translations) organization.

Here I mention to let you know about this initiative:
@hootlex @danielroe @danielkellyio @antfu @lmiller1990 @NataliaTepluhina 
@yyx990803 @posva @shentao @sdras @pikax @LinusBorg @nickmessing 
@kiaking @kazupon @Jinjiang @wxsms @veaba @jay-es

**Comment by skirtles-code** [comment]
Thanks. I suggest posting the details of your translation in the Discussion section of <https://github.com/vuejs-translations/guidelines/discussions>. It seems that you already started a thread back in February, but it doesn't currently include the repo details: <https://github.com/vuejs-translations/guidelines/discussions/26>.

**Reply by nazarepiedady** [reply]
@skirtles-code yes, you have the point, I already knew that 
but happened that I did not pay attention to some points 
of Vue.js Translations Guidelines and forget that 
the announcement should be done there too,
 hehe 😅, it is all my fault.
